lessons/08-programming-grammar/notes/values-types.md

```diff
@@ -6,10 +6,13 @@ A _value_ can be most anything, like a name (`"Jeremy"`), a whole number (`5000`
 Every _value_ can be described by a _type_. There are lots of _types_ in Ruby, and we are going to talk about three of them today: `String`, `Fixnum`, and `Float`.
 
 ### Types
+
+#### String
 A `String` is any sequence of letters and/or numbers enclosed in either single (`'`) or double (`"`) quotes. `"Elephant Hotdog"` is a `String`, and so is `'42'`. Most of the time, you can use either single or double quotes to create a _string value_.
 
+#### Fixnum
 A `Fixnum` is a whole number expressed without a decimal. There's a couple ways to create _fixnum values_, but the most common is to type out the number. `1` is a `Fixnum`, and so is `23` and `-4500`.
 
 #### Question: How are `42` and `"42"` different?
```

Comment:
I like this question, but I don't think the question is answered below. Also the stuff about float below is missing the #### Float heading